contracts/test/Summa.ts

```diff
@@ -421,19 +380,20 @@ describe("Summa Contract", () => {
       leafHash = inclusionCalldata.public_inputs[0];
       mstRoot = inclusionCalldata.public_inputs[1];
       inclusionProof = inclusionCalldata.proof;
+      rootSum = BigNumber.from(10000000);
     });
 
     it("should verify the proof of inclusion for the given public input", async () => {
       await summa.submitProofOfAddressOwnership(ownedAddresses);
-      await submitProofOfSolvency(summa, mstRoot, solvencyProof);
+      await submitCommitment(summa, mstRoot, rootSum);
```

Comment:
I realize that there's a security issue here. You hardcode `rootSum` as 10000000 when sending the commitment. But this is not the correct root sum associated with `mstRoot`. To fix that, the `ProofOfInclusion` should also output the `rootSum` as public output and the verification function should check that it matches the committed one